client/asset/dcr/dcr.go

```diff
@@ -78,7 +83,18 @@ var (
 		{
 			Key:         "account",
 			DisplayName: "Account Name",
-			Description: "dcrwallet account name",
+			Description: "dcrwallet account name, should be the mixed account if mixing is enabled on the wallet",
+		},
+		{
+			Key:         "unmixedaccount",
+			DisplayName: "Change Account Name",
+			Description: "dcrwallet change account name, if mixing is enabled on the wallet",
+		},
+		{
+			Key:         "tradingaccount",
+			DisplayName: "Dedicated Trading Account",
```

Comment:
> "dcrwallet account to temporarily store change from chained swaps in multi-lot orders"

Almost perfect. But this will also hold split tx outputs that are not change outputs. So, I'm unsure if the name "Trade Change" account or suggested description would be less confusing. Will think about this more.

> I also wonder if we should attempt to create this account if it does not exist on connect.

Thought about this but wondered if it's better for the user to create the account so they have a better understanding of the account's purpose.